### PR TITLE
Migrate Product details to new MacawUI

### LINKED ISSUE #3353: Product Details

### PR DESCRIPTION
**Pull request by krzysztofzuraw**
I want to merge this change because it fixes #3353 

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [x] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation
22. [ ] variants
23. [ ] payments

CONTAINERS=6

### CODE REVIEW

File: src/components/AppLayout/TopNav/Menu.tsx [hunk 1]
```diff
@@ -40,7 +37,7 @@ export const TopNavMenu: React.FC<TopNavMenuProps> = ({
           backgroundColor="surfaceNeutralPlain"
         >
           {items.map(item => (
-            <Dropdown.Item key={item.label}>
```
Comment (poulch) [inline src/components/AppLayout/TopNav/Menu.tsx:43]: question: why ? Do we need key in map?

File: src/components/Attributes/BasicAttributeRow.tsx [hunk 2]
```diff
@@ -1,52 +1,53 @@
...
+      data-test-id="attribute-label"
+      as="label"
+      htmlFor={id}
+      display="flex"
+      gap={3}
+      cursor={clickableLabel ? "pointer" : "auto"}
+    >
+      <Text>{label}</Text>
+      {description && (
+        <Tooltip title={description} placement="top">
```
Comment (poulch) [inline src/components/Attributes/BasicAttributeRow.tsx:39]: suggestion: you can use new tooltip from macaw

File: src/components/ChannelsAvailabilityCard/Channel/ChannelAvailabilityItemContent.tsx [hunk 3]
```diff
@@ -141,64 +141,75 @@ const ChannelContent: React.FC<ChannelContentProps> = ({
...
+            <Box
+              display="flex"
+              gap={3}
+              flexDirection="column"
+              alignItems="start"
+            >
+              <Checkbox
+                onCheckedChange={(checked: boolean) =>
+                  setAvailableDate(checked)
+                }
```
Comment (poulch) [inline src/components/ChannelsAvailabilityCard/Channel/ChannelAvailabilityItemContent.tsx:208]:
```suggestion
                onCheckedChange={setAvailableDate}
```

**Review by poulch — APPROVED** [review]

**Review by andrzejewsky — DISMISSED** [review]

**Review by poulch — APPROVED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Comment by krzysztofzuraw** [comment]
Blocked by https://github.com/saleor/macaw-ui/issues/384

**Review by poulch — APPROVED** [review]

**Review by poulch — APPROVED** [review]

**Review by wojteknowacki — DISMISSED** [review]

**Review by poulch — DISMISSED** [review]

**Review by andrzejewsky — DISMISSED** [review]

**Review by wojteknowacki — APPROVED** [review]

**Review by poulch — APPROVED** [review]

**Pull request merged**